File: scratch/search_transcript.js
const fs = require('fs');
const readline = require('readline');
const path = require('path');

const logPath = 'C:\\Users\\pc\\.gemini\\antigravity\\brain\\7f6a0bb8-1272-4502-a25f-41f870c067dd\\.system_generated\\logs\\transcript.jsonl';

async function run() {
  const fileStream = fs.createReadStream(logPath);
  const rl = readline.createInterface({
    input: fileStream,
    crlfDelay: Infinity
  });

  console.log("=== Matching Lines ===");
  for await (const line of rl) {
    if (line.toLowerCase().includes("gol") || line.toLowerCase().includes("simül") || line.toLowerCase().includes("asist") || line.toLowerCase().includes("ver")) {
      try {
        const obj = JSON.parse(line);
        if (obj.type === 'USER_INPUT') {
          console.log(`[USER] Step ${obj.step_index}: ${obj.content}`);
        } else if (obj.type === 'PLANNER_RESPONSE' && obj.content) {
          // print snippet of assistant response
          console.log(`[AGENT] Step ${obj.step_index}: ${obj.content.substring(0, 150)}...`);
        }
      } catch (e) {
        // ignore parse error
      }
    }
  }
}
run();

File: scratch/search_transcript.py
import json

log_path = r"C:\Users\pc\.gemini\antigravity\brain\d920dd82-04c6-4a2d-80e5-0fae7134eefc\.system_generated\logs\transcript.jsonl"

with open(log_path, "r", encoding="utf-8") as f:
    for line in f:
        try:
            data = json.loads(line)
            if data.get("type") == "USER_INPUT":
                content = data.get("content", "")
                # Print user message if it mentions team or scores
                if any(word in content.lower() for word in ["skor", "tarih", "saat", "mac", "maç", "fransa", "paraguay", "türkiye", "turkey"]):
                    print(f"[{data.get('step_index')}]: {content}\n")
        except Exception as e:
            pass
